execution/gethexec/node.go

```diff
@@ -214,7 +214,12 @@ var ConfigDefault = Config{
 	ExposeMultiGas:              false,
 }
 
-type ConfigFetcher func() *Config
+type ConfigFetcher interface {
+	Get() *Config
+	Start(context.Context)
+	StopAndWait()
+	Started() bool
+}
```

Comment:
if I'm not mistaken, we don't need any of `Start`, `StopAndWait` or `Started`; if that's the case, we can simply stay with a `func() *Config` alias here and avoid changes in `cmd/nitro/nitro.go`